aws/resource_aws_ebs_volume.go

```diff
@@ -373,3 +371,40 @@ func resourceAwsEbsVolumeDelete(d *schema.ResourceData, meta interface{}) error
 
 	return nil
 }
+
+func resourceAwsEbsVolumeCustomizeDiff(_ context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	iops := diff.Get("iops").(int)
```

Comment:
Should `throughput` be validated here as well? It's only valid if the volume type is `gp3`

Reply:
Added plan-time check; Thanks.